src/runtime/test/host_port_test.cpp

```diff
@@ -248,7 +247,7 @@ TEST(host_port_test, dns_resolver)
         host_port hp2("localhost", 8081);
         g_hp->set_leader(hp2);
 
-        auto addr_grp = resolver.resolve_address(hp_grp);
+        auto addr_grp = dns_resolver::instance().resolve_address(hp_grp);
         auto g_addr = addr_grp.group_address();
```

Comment:
`rpc_group_address::count()` could also be declared as `const`:
```suggestion
        const auto * const g_addr = addr_grp.group_address();
```